docs/root/intro/version_history.rst

```diff
@@ -5,6 +5,7 @@ Version history
 ================
 * config: use type URL to select an extension whenever the config type URL (or its previous versions) uniquely identify a typed extension, see :ref:`extension configuration <config_overview_extension_configuration>`.
 * retry: added a retry predicate that :ref:`rejects hosts based on metadata. <envoy_api_field_route.RetryPolicy.retry_host_predicate>`
+* string matcher: added :ref:`ignore_case <envoy_api_field_type.matcher.StringMatcher.ignore_case>` that allows case insensitive matching.
```

Comment:
I would drop this, we don't generally document things like this in release notes.

Reply:
reverted.